### DISCUSSION TITLE
In-game GUI for Playtime Reward

### DISCUSSION THREAD

**Discussion by YusakiDev** [Zrips/CMI#4696] [Feature Requests]
**Description of feature/improvement request:**
- Create Gui editor for playtime reward in-game 
---
### Detailed explanation of what it should do, when and in what situation: ###
it should be able to open UI for players who type /prewards
it should be able to open UI for admin who type /prewardedit 
/preawrdedit command should open GUI for admin to edit player playtime reward UI and be able to create new or edit existing playtime reward 
### Basic explanation as to why this feature is needed and/or useful to have: ###
It's the thing that all the server should have as GUI will look better than a clickable text message 
and It will be easier to create playtime rewards in-game using in-game commands

**Comment by MAKROFREEK** [comment]
Heck I'd even settle for placeholders so I can make my own GUI in deluxemenus or something similar.

**Comment by TomLewis** [comment]
I already requested this.
https://github.com/Zrips/CMI/discussions/7575
